system/websocket/Make.defs

```diff
@@ -0,0 +1,26 @@
+############################################################################
+# apps/system/cwebsocket/Make.defs
```

Comment:
should we move to netutils folder? since all network related library put there.

Reply:
sure